lib/internal/util.js

```diff
@@ -53,6 +53,16 @@ const experimentalWarnings = new SafeSet();
 
 const colorRegExp = /\u001b\[\d\d?m/g; // eslint-disable-line no-control-regex
 
+
+let uvBinding;
+
+function lazyUv() {
+  if (!uvBinding) {
+    uvBinding = internalBinding('uv');
+  }
+  return uvBinding;
```

Comment:
```suggestion
  return uvBinding ??= internalBinding('uv');
```
if we are willing to mix assignments with return statements

Reply:
@RaisinTen I think assignment and return should not be mixed